pkgs/by-name/ra/raffi/package.nix

```diff
@@ -8,16 +8,16 @@
 
 rustPlatform.buildRustPackage rec {
```

Comment:
Good call - thanks. I'm going to send a PR for the fix since I can't amend this PR.

Reply:
Don't use `assert`. `meta.broken` is okay.
But I don't see why it's needed if the fuzzel version in master is new enough.

Reply:
I learned something new today :-) I'm okay with either one, or neither since the version of fuzzel will always be correct if taking both directly from the same channel branches.

Reply:
I created a separate PR using `meta.broken`: https://github.com/NixOS/nixpkgs/pull/342604 Feel free to choose either this one or that one.